lab-cameron/lib/fs-readfile.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const fs = require('fs');
+
+// I LEFT IN THESE CONSOLE LOGS TO GIVE MY TESTS MORE CONTEXT AND AS PROOF OF ASYNCHRONICITY //
+
+module.exports = function(paths) {
+  let buffers = [];
+  fs.readFile(paths[0], function(err, data) {
+    if (err) throw err;
+    console.log('utf-8 format: ', data.toString('utf-8', 0, 8));
+    buffers.push(data.toString('hex', 0, 8));
+
+    fs.readFile(paths[1], function(err, data) {
+      if (err) throw err;
+      console.log('utf-8 format: ', data.toString('utf-8', 0, 8));
+      buffers.push(data.toString('hex', 0, 8));
+
+      fs.readFile(paths[2],function(err, data) {
+        if (err) throw err;
+        console.log('utf-8 format: ', data.toString('utf-8', 0, 8));
+        buffers.push(data.toString('hex', 0, 8));
+      });
+    });
+  });
+
+  return buffers;
```

Comment:
Also, I have no idea why I needed to do ``if (callback) return callback(buffers)``. It should have just worked with a normal ``callback(buffers)``, but I had to do this hacky workaround to get it to work.